default.py

```diff
@@ -11,6 +11,11 @@
 from operator import itemgetter
 
 import requests
+import requests.packages.urllib3
+
+import cookielib
```

Comment:
As I said before, I have never worked with cookies. So if it turns out that you need cookielib to get it working, so be it. Just wanted to make sure you didn't know about cookies in requests.